packages/x-data-grid/src/internals/utils/propValidation.ts

```diff
@@ -1,3 +1,4 @@
+import { isNumber } from '@mui/x-data-grid/utils/utils';
```

Comment:
We can't do two level imports, nor can we important from the same npm package.

Reply:
I had imports like this added by IntelliSense too 😅
Recently the IntelliSense import suggestions experience has worsened for me in MUI X.
Not sure if this was caused by some change on our side.

I'll try to add an ESLint rule for this.

Reply:
https://github.com/mui/mui-x/pull/12477